src/SecTester.Reporter/DefaultFormatter.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using SecTester.Scan.Models;
+
+namespace SecTester.Reporter;
+
+internal class DefaultFormatter : Formatter
+{
+  public const char NewLine = '\n';
+  private const char BulletPoint = '●';
+  private const char Tabulation = '\t';
+  private const string TemplateBody = @"Issue in Bright UI:   {0}
+Name:                 {1}
+Severity:             {2}
+Remediation:
+{3}
+Details:
+{4}";
+  private const string TemplateExtraDetails = "\nExtra Details:\n{5}";
+  private const string TemplateReferences = "\nReferences:\n{6}";
+
+  private static readonly IEnumerable<Comment> EmptyComments = new List<Comment>();
+  private static readonly IEnumerable<string> EmptyResources = new List<string>();
+
+  public string Format(Issue issue)
+  {
+    var comments = issue.Comments ?? EmptyComments;
+    var resources = issue.Resources ?? EmptyResources;
+
+    var template = GenerateTemplate(comments.Any(), resources.Any());
+
+    var message = string.Format(CultureInfo.InvariantCulture,
+      template,
+      issue.Link,
+      issue.Name,
+      issue.Severity,
+      issue.Remedy,
+      issue.Details,
+      FormatList(comments, FormatExtraInfo),
+      FormatList(resources)
+    );
+
+    return message.Trim().Replace(Environment.NewLine, NewLine.ToString());
+  }
+
+  private static string GenerateTemplate(bool extraInfo, bool references)
+  {
+    var stringBuilder = new StringBuilder(TemplateBody);
+
+    if (extraInfo)
+    {
+      stringBuilder.Append(TemplateExtraDetails);
+    }
+
+    if (references)
+    {
+      stringBuilder.Append(TemplateReferences);
+    }
+
+    return stringBuilder.ToString();
+  }
+
+  private static string FormatExtraInfo(Comment comment)
+  {
+    var footer = comment.Links is not null && comment.Links.Any() ? CombineList(comment.Links.Prepend("Links:")) : String.Empty;
```

Comment:
Use `""` instead of String.Empty

```suggestion
    var footer = comment.Links is not null && comment.Links.Any() ? CombineList(comment.Links.Prepend("Links:")) : "";
```